mu4/userscores/qml/MuseScore/UserScores/internal/ScoreItem.qml

```diff
@@ -80,21 +142,26 @@ Item {
         Rectangle {
             anchors.fill: parent
 
-            color: "#FFFFFF"
+            color: "white"
```

Comment:
I'd prefer to leave it as HEX

Reply:
I prefer readability, so symbolic names, whenever possible